html/manifest/display_override.json

```diff
@@ -0,0 +1,55 @@
+{
+  "html": {
+    "manifest": {
+      "display_override": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/Manifest/display_override",
+          "spec_url": "https://wicg.github.io/manifest-incubations/#display_override-member",
+          "support": {
+            "chrome": {
+              "version_added": "89"
+            },
+            "chrome_android": {
+              "version_added": false
+            },
+            "edge": {
+              "version_added": "89"
+            },
+            "firefox": {
+              "version_added": false
+            },
+            "firefox_android": {
+              "version_added": false
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": null
+            },
+            "opera_android": {
+              "version_added": false
+            },
+            "safari": {
+              "version_added": false
+            },
+            "safari_ios": {
+              "version_added": false
+            },
+            "samsunginternet_android": {
+              "version_added": false
```

Comment:
The latest Chromium engine the [release notes](https://developer.samsung.com/internet/release-note.html) mention is still Chromium 87.4280, so `false` might actually be correct. @torgo to verify.